src/components/layout.js

```diff
@@ -25,21 +26,18 @@ const Layout = ({ children }) => (
     render={data => (
       <>
         <Header siteTitle={data.site.siteMetadata.title} />
-        <div
-          style={{
-            margin: `0 auto`,
-            maxWidth: 960,
-            padding: `0px 1.0875rem 1.45rem`,
-            paddingTop: 0,
-          }}
-        >
-          <main>{children}</main>
-          <footer>
+
+        <Box m="0 auto" maxWidth={960} px={1}>
+          <Box is="main">{children}</Box>
```

Comment:
`is` prop is not working or maybe deprecated na

Reply:
oops my bad it should be `as`